test/e2e_node/image_volume.go

```diff
@@ -155,4 +159,41 @@ var _ = SIGDescribe("ImageVolume", nodefeature.ImageVolume, func() {
 		output := e2epod.ExecCommandInContainer(f, podName, containerName, "/bin/ls", filepath.Dir(volumePathPrefix))
 		gomega.Expect(output).NotTo(gomega.ContainSubstring(strings.TrimPrefix(volumePathPrefix, "/")))
 	})
+
+	f.It("should succeed with multiple pods and same image", func(ctx context.Context) {
```

Comment:
Are those tests always running on a single node?

Reply:
In my opinion, all tests under the `e2e-node` directory must be running on the cluster with a single node. `e2e` tests may run on a cluster with multiple nodes.

Reply:
It can have multiple nodes, according to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md 

I have to add a node selector to the pod spec to make sure the pod is scheduled on the same node.

Reply:
Sounds good, thank you for double checking!